### ISSUE TITLE
[feature request] Real Debrid (Add torrents to queue?)

### ISSUE THREAD

**Issue by minicoz** [rogerfar/rdt-client#633]
**What version are you using?**
 v2.0.86. 

**Wat OS are you running?**
Docker

**Are you using Docker or as a service?**
Yes

**Which debrid provider are you using?**
Real Debrid

**Which downloader are you using?**
Internal

So Real Debrid now only allows 10 concurrent torrent downloads at the same time. I use rdt-client as a downloader from prowlarr for collections of torrents. Could you implement some sort of queue system so that 10 torrents could be tried at time and then as they finished or expire, the queue would automatically load the new batch of torrent files? 

I was able to send all 50 torrents to rdt client and even though real debrid had a limit, I was able to at least start all of them and slowly remove the finished ones. Now its limiting everything to only 10 torrents at a time. 

Love your work, boa noches.

**Comment by JRDevo** [comment]
Tried to grab a complete series today and had this issue.

**Comment by skijbal** [comment]
This would be nice so you can queue up a list and not have to worry about RD not grabbing them.

**Comment by JRDevo** [comment]
Feels more like a bug then feature tbh.... Id like to download more hen 10 episodes quite often

**Comment by JRDevo** [comment]
agreed, its a bug. Especially important if you want to start downloading an old series.
